### PR TITLE
Change file layouts folder

### PR DESCRIPTION
**Pull request by annq3012**
## Description
Change left-bar and header get information of user login
## Reference Requirements
- Product Backlog ID: PBI-1
- Task ID: TI-1

## Reference Isuees
#{issue_id}

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

File: resources/views/backend/layouts/partials/left-bar.blade.php [hunk 1]
```diff
@@ -52,9 +52,9 @@
...
         </a>
       </li>
-      <li>
+      <li class="{{ areActiveRoute(['posts.index', 'posts.edit', 'posts.show']) }}">
         <a href="#">
-          <i class="fa fa-list" aria-hidden="true"></i>
+          <i class="fa fa-id-card-o" aria-hidden="true"></i>
           <span>{{ __('dashboard.posts') }}</span>
           <span class="pull-right-container">
               <small class="label pull-right bg-purple">{{ getCount(App\Model\Post::class) }}</small>
```
Comment (at-anhngo) [inline resources/views/backend/layouts/partials/left-bar.blade.php]: 4 spaces

File: resources/views/backend/layouts/partials/header.blade.php [hunk 2]
```diff
@@ -11,20 +11,20 @@
...
           </a>
           <ul class="dropdown-menu">
             <li class="user-header">
-              <img src="{{ asset('bower_components/admin-lte/dist/img/user2-160x160.jpg') }}" class="img-circle" alt="User Image">
+              <img src="{{ Auth::user()->avatar_url }}" class="img-circle" alt="User Image">
               <p>
-                Alexander Pierce - {{__('Web Developer')}}
-                <small>{{__('Member since Jun. 2017')}}</small>
+                {{ Auth::user()->name }} - {{__('dashboard.web_developer')}}
+                <small>{{__('dashboard.member')}} : {{ date('d-m-Y', strtotime(Auth::user()->created_at)) }}</small>
```
Comment (at-anhngo) [inline resources/views/backend/layouts/partials/header.blade.php]: nên sử dụng Carbon để format date

**Review by ghost — CHANGES_REQUESTED** [review]
Check my comment

File: resources/views/backend/layouts/partials/header.blade.php [hunk 3]
```diff
@@ -11,20 +11,21 @@
...
-                <small>{{__('Member since Jun. 2017')}}</small>
+                {{ Auth::user()->name }} - {{__('dashboard.web_developer')}}
+                {{-- {{dd(Auth::user()->created_at)}} --}}
+                <small>{{__('dashboard.member')}} : {{ Carbon\Carbon::parse(Auth::user()->created_at)->format('d-m-Y')}}</small>
               </p>
             </li>
             <li class="user-footer">
               <div class="pull-left">
-                <a href="#" class="btn btn-default btn-flat">{{__('Profile')}}</a>
+                <a href="{{route('users.show', Auth::user()->employee_code)}}" class="btn btn-default btn-flat">{{__('Profile')}}</a>
```
Comment (at-anhngo) [inline resources/views/backend/layouts/partials/header.blade.php]: nên bỏ vào 1 file lang

**Review by ghost — CHANGES_REQUESTED** [review]
check my comment

**Comment by at-anhngo** [comment]
fix circle ci

**Comment by at-anhngo** [comment]
LGTM

**Review by ghost — APPROVED** [review]

File: app/Helpers/common.php [hunk 4]
```diff
@@ -38,3 +38,39 @@ function getCount($name)
...
+
+    /**
+     * Active menu side bar when routes menu are current route
+     *
+     * @param Array  $routes routes action
+     * @param string $output active or ''
+     *
+     * @return string
+     */
+    function areActiveRoute(array $routes, $output = "active")
```
Comment (at-duyho) [inline app/Helpers/common.php]: - tại sao lại dùng cả hai hàm cho một chức năng?

**Review by ghost — CHANGES_REQUESTED** [review]
check my comment

**Review by ghost — CHANGES_REQUESTED** [review]
check my comment

**Comment by at-duyho** [comment]
fix CI

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**